pkg/virt-launcher/virtwrap/statsconv/converter.go

```diff
@@ -99,14 +99,25 @@ func Convert_libvirt_MemoryStat_to_stats_DomainStatsMemory(inMem []libvirt.Domai
 
 func Convert_libvirt_DomainStatsVcpu_To_stats_DomainStatsVcpu(in []libvirt.DomainStatsVcpu) []stats.DomainStatsVcpu {
 	ret := make([]stats.DomainStatsVcpu, 0, len(in))
+	humanReadableState := ""
 	for _, inItem := range in {
+		switch inItem.State {
```

Comment:
Can we move this conversion to `updateVcpu` and remove it `DomainStatsVcpu` as I think it was intended to reflect libvirt structs.

Reply:
done

also added some unit tests for the conversion

Reply:
oops, just noticed my tests are failing... not at home right now, I'll take a look by the weekend

Reply:
it should be fine now